src/include/btmem.h

```diff
@@ -636,7 +636,11 @@ struct __wt_page_block_meta {
     uint64_t checkpoint_id;
     uint64_t reconciliation_id;
 
-    /* TODO: Place additional metadata here, e.g., checkpoint ID, number of deltas. */
+    uint64_t backlink_checkpoint_id;
+    uint64_t base_checkpoint_id;
+    uint32_t delta_count;
+    uint64_t disagg_lsn;
+    uint64_t disagg_lsn_frontier;
```

Comment:
How are we using this field? I thought LSN frontier is just one global field per WT connection.

Reply:
Yes, but the lsn frontier value is returned by the PALI interface - and some component will need to update the global based on the returned value.  I guess I didn't see the block manager doing this, so maybe the btree, which may need to know something about lsns.

Reply:
It doesn't belong with the page, and it shouldn't be returned by `plh_get` either. Getting the LSN frontier should be a separate PALI call. I recommend we remove this field both from here and from get arguments, and then add it whenever we get to the point of defining the appropriate interface.

Reply:
Okay, I removed it.